### DISCUSSION TITLE
Privacy Manifest

### DISCUSSION THREAD

**Discussion by chrs1885** [pointfreeco/swift-url-routing#91] [Q&A]
Hi there,

We recently became aware that, according to the new Apple privacy regulations, third-party SDKs will also be required to provide a Privacy Manifest to ensure compliance with privacy policies in the future.

Do you have any plans to deliver a Privacy Manifest with the library?

**Comment by stephencelis** [comment]
@chrs1885 Do you have an example of another Swift package out there that does this? I think privacy manifests may only be necessary for binary frameworks, and I don't even think Apple's packages contain privacy manifests.

Since this isn't a bug with the library I'm going to convert to a discussion, but if there is precedent for packages out there with privacy manifests, we'd take a PR to include one.

**Reply by chrs1885** [reply]
@stephencelis according to the docs also swift packages can provide a Privacy Manifest: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files.

**Reply by stephencelis** [reply]
@chrs1885 I've seen the docs, but I haven't found much evidence that a typical SPM package requires a privacy manifest. Folks are speculating that it mostly seems necessary for binaries distributed via SPM.

Can you find examples of folks using privacy manifests in their SPM packages? Even Apple doesn't seem to do so. Perhaps it'd be good to open an issue on one of their repos to get a more official stance?
